examples/plot_tikhonov.py

```diff
@@ -0,0 +1,229 @@
+                  n_spatial_basis=n_spatial_basis,
+                  n_temporal_basis=n_temporal_basis)
+spatial_basis = strf_model.make_gaussian_basis()
+strf_model.visualize_gaussian_basis(spatial_basis)
+
+########################################################
+# Design temporal basis functions
+
+time_points = np.linspace(-100., 100., 10.)
+centers = [-75., -50., -25., 0, 25., 50., 75.]
+temporal_basis = strf_model.make_raised_cosine_temporal_basis(time_points=time_points,
+                                                              centers=centers,
+                                                              widths=10.*np.ones(7))
+plt.plot(time_points, temporal_basis)
+plt.show()
+
+########################################################
+# Design parameters
+
+# Spatial extent
+n_shape = 50
+n_features = n_spatial_basis
+
+# Window of interest
+window = [-100, 100]
+
+# Bin size
+binsize = 20
+
+# Zero pad bins
+n_zero_bins = np.floor((window[1] - window[0]) / binsize / 2)
+
+########################################################
+# Build design matrix
+
+bin_template = np.arange(window[0], window[1] + binsize, binsize)
+n_bins = len(bin_template) - 1
+
+probetimes = probes_df['t_probe'].values
+spatial_features = np.zeros((0, n_features))
+spike_counts = np.zeros((0,))
+fixation_id = np.zeros((0,))
+
+# For each fixation
+for fx in fixations_df.index[:1000]:
+
+    # Fixation time
+    fixation_time = fixations_df.loc[fx]['t_fix_f']
+
+    this_fixation_spatial_features = np.zeros((n_bins, n_spatial_basis))
+    this_fixation_spikecounts = np.zeros(n_bins)
+    unique_fixation_id = fixations_df.loc[fx]['trialNum_f'] + 0.01 * fixations_df.loc[fx]['fixNum_f']
+    this_fixation_id = unique_fixation_id * np.ones(n_bins)
+
+    # Look for probes in window of interest relative to fixation
+    probe_ids = np.searchsorted(probetimes,
+                                [fixation_time + window[0] + 0.1,
+                                 fixation_time + window[1] - 0.1])
+
+    # For each such probe
+    for probe_id in range(probe_ids[0], probe_ids[1]):
+
+        # Check if probe lies within spatial region of interest
+        fix_row = fixations_df.loc[fx]['y_curFix_f']
+        fix_col = fixations_df.loc[fx]['x_curFix_f']
+        probe_row = probes_df.loc[probe_id]['y_probe']
+        probe_col = probes_df.loc[probe_id]['x_probe']
+
+        if ((probe_row - fix_row) > -n_shape / 2 and
+            (probe_row - fix_row) < n_shape / 2 and
+            (probe_col - fix_col) > -n_shape / 2 and
+            (probe_col - fix_col) < n_shape / 2):
+
+            # Get probe timestamp relative to fixation
+            probe_time = probes_df.loc[probe_id]['t_probe']
+            probe_bin = np.where(bin_template < (probe_time - fixation_time))[0][-1]
+
+            # Define an image based on the relative locations
+            img = np.zeros(shape=(n_shape, n_shape))
+            row = -np.round(probe_row - fix_row) + n_shape/2 - 1
```

Comment:
`n_shape / 2`. Are we following pep8? I am not actually sure because I just saw your last commit which seems to suggest we are, but I don't see pep8 compatible code here. Are the examples somehow ignored by the pep8 checker?